chucks-pick3-mobile/gulpfile.js

```diff
@@ -0,0 +1,32 @@
+const { src, dest, task } = require("gulp"),
+  clean = require("gulp-clean"),
+  gutil = require("gulp-util");
+
+task("remove-mocks", function () {
+  return src(["src/**/*mock*.*"], { read: false }).pipe(clean());
+});
+
+task("include-mocks", function (done) {
+  // the base option sets the relative root for the set of files,
+  // preserving the folder structure
+  console.log("inside inlcude-mocks gulp tasks.");
+  if (process.env.USE_MOCKS === "true") {
+    src(["test/mocks/**/*.*", "test/environments/**/*.*"]).pipe(dest("src"));
+  }
+
+  done();
+});
+
+task("remove-prod", function () {
+  return src(["src/**/*prod*.*"], { read: false }).pipe(clean());
+});
+task("include-prod", function (done) {
+  // the base option sets the relative root for the set of files,
+  // preserving the folder structure
+  console.log("inside inlcude-mocks gulp tasks.");
+  if (process.env.USE_MOCKS === "true") {
+    src(["test/prod/**/*.*", "test/environments/**/*.*"]).pipe(dest("src"));
+  }
+
```

Comment:
We need to create a gulp task that run the ng serve command and dynamically adds the `--configuration=mock` parameter if `process.env.USE_MOCKS === "true"`

It should do the same thing as original start script: `" ionic serve --configuration=mock"`

See if you can tell gulp task to run the `" ionic serve --configuration=mock"` command.

Reply:
Completed work on gulp task.